[DEV] Code submission "lint_and_test" fails with error ERR_PNPM_OTHER_PM_EXPECTED

**Description**
Submitting a new PR causes the GitHub workflow "lint_and_test" to execute.

This fails during the "Install Dependencies" step with error:
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/d7930cbe-ed5e-4cc9-bbb1-30df5be0c13c)

```
 ERR_PNPM_OTHER_PM_EXPECTED  This project is configured to use npm
```

--- a/.github/workflows/e2e.yml
+++ b/.github/workflows/e2e.yml
@@ -6,6 +6,7 @@ on: workflow_dispatch
 
 env:
     OBSIDIAN_VERSION: 1.2.8
+    COREPACK_ENABLE_STRICT: 0
 
 jobs:
     e2e:

--- a/.github/workflows/pr.yml
+++ b/.github/workflows/pr.yml
@@ -4,6 +4,9 @@ on:
     pull_request:
         branches: [master]
 
+env:
+    COREPACK_ENABLE_STRICT: 0
+
 jobs:
     lint_and_test:
         runs-on: ubuntu-latest

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -7,6 +7,7 @@ on:
 
 env:
     PLUGIN_NAME: obsidian-spaced-repetition
+    COREPACK_ENABLE_STRICT: 0
 
 jobs:
     release:

--- a/package.json
+++ b/package.json
@@ -50,5 +50,5 @@
         "chart.js": "^4.4.2",
         "pagerank.js": "^1.0.2"
     },
-    "packageManager": "npm@8.18.0"
+    "packageManager": "^pnpm@8.15.4"
 }

